action.yml

```diff
@@ -17,6 +20,16 @@ inputs:
   port:
     description: "The port to use for the application"
     required: false
+  registry:
```

Comment:
Not sure about this, may be `registry_server` sounds better?

Reply:
I tend to see "hostname" used for this. `registry_hostname`.

Reply:
Okay :+1: